src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -215,16 +224,6 @@ void checkSendOfflineQAEmail(StageWithCaseData stage) {
         }
     }
 
-    String getOfflineQaUser(String stageData) {
```

Comment:
We don't need this method to turn the data string into a map to get one field out any more.